examples/oft-aptos/aptos.layerzero.config.ts

```diff
@@ -0,0 +1,161 @@
+import { EndpointId } from '@layerzerolabs/lz-definitions-v3'
+import { ExecutorOptionType } from '@layerzerolabs/lz-v2-utilities-v3'
+import { bscConfig } from './evmOAppConfig'
+import { ethConfig } from './evmOAppConfig'
+
+import type { OAppOmniGraphHardhat, OmniPointHardhat } from '@layerzerolabs/toolbox-hardhat'
+
+const ethereumContract: OmniPointHardhat = {
+    eid: EndpointId.ETHEREUM_V2_TESTNET,
+    contractName: 'MyOFT',
+}
+
+const bscContract: OmniPointHardhat = {
+    eid: EndpointId.BSC_V2_TESTNET,
+    contractName: 'MyOFT',
+}
+
+const aptosContract: OmniPointHardhat = {
+    eid: EndpointId.APTOS_V2_TESTNET,
+    contractName: 'oft',
+}
+
+const ethereumSandboxContract: OmniPointHardhat = {
```

Comment:
I think we should remove all sandbox related information from devtools. Devtools is publicly facing and in other examples we only use testnets AFAIK.